src/confluent_kafka/src/AdminTypes.c

```diff
@@ -147,9 +162,15 @@ static PyMemberDef NewTopic_members[] = {
 
 
 static PyObject *NewTopic_str0 (NewTopic *self) {
-        return cfl_PyUnistr(
+        if(self->num_partitions != -1){
```

Comment:
Better to have check for exception instead of normal case.

Also, style. Space between ) and {